src/styles/stars.scss

```
@@ -0,0 +1,33 @@
+.card__stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  height: 16px;
+  color: #060B35;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
```

```suggestion
    background: url(../images/star.svg) no-repeat center;
    height: 16px;
    width: 16px;
```

can be a bit simplified